scripts/run-actv-3-rdf.py

```diff
@@ -0,0 +1,46 @@
+#Checks if all declared prefixes are used in the RDF File
```

Comment:
Can you change the file name according this naming convention? https://github.com/naming-convention/naming-convention-guides/tree/master/python